packages/cli/src/commands/pw-test.ts

```diff
@@ -0,0 +1,365 @@
+      return {
+        logicalId: `playwright-check-${inputLogicalId}`,
+        name: `Playwright Test: ${input}`,
+        testCommand,
+        locations: [runLocation],
+        frequency: 10,
+      }
+  }
+
+  private getConfigPath (playwrightFlags: string[]) {
+    for (let i = 0; i < playwrightFlags.length; i++) {
+      const arg = playwrightFlags[i]
+      if (arg.startsWith('--config') || arg.startsWith('-c')) {
+        return arg.includes('=') ? arg.split('=')[1] : playwrightFlags[i + 1]
+      }
+    }
+  }
+
+  private async createPlaywrightCheck (playwrightCheck: PlaywrightSlimmedProp, playwrightConfigPath: string = './playwright.config.ts') {
+    const dir = process.cwd()
+    const baseName = path.basename(dir)
+
+    const configFile = await getChecklyConfigFile()
+    if (!configFile) {
+      this.style.shortWarning('No Checkly config file found')
+      this.style.shortInfo('Creating a default checkly config file.')
+      const checklyConfig = getDefaultChecklyConfig(baseName, `./${path.relative(dir, playwrightConfigPath)}`, playwrightCheck)
+      await writeChecklyConfigFile(dir, checklyConfig)
+      this.style.actionSuccess()
+      return
+    }
+    const checklyAst = recast.parse(configFile.checklyConfig)
+    const checksAst = findPropertyByName(checklyAst, 'checks')
+    if (!checksAst) {
+      this.style.longError('Unable to automatically sync your config file.', 'This can happen if your Checkly config is ' +
+        'built using helper functions or other JS/TS features. You can still manually set Playwright config values in ' +
+        'your Checkly config: https://www.checklyhq.com/docs/cli/constructs-reference/#project')
+
+      return
+    }
+    const b = recast.types.builders;
+    const playwrightPropertyNode = b.property(
+      'init',
+      b.identifier('playwrightConfigPath'),
+      b.stringLiteral(playwrightConfigPath)
+    );
+
+    const playwrightCheckString = `const playwrightCheck = ${JSON5.stringify(playwrightCheck, { space: 2 })}`
+    const playwrightCheckAst = recast.parse(playwrightCheckString)
+    const playwrightCheckNode = playwrightCheckAst.program.body[0].declarations[0].init;
+    addOrReplaceItem(checksAst.value, playwrightPropertyNode, 'playwrightConfigPath')
+    addItemToArray(checksAst.value, playwrightCheckNode, 'playwrightChecks')
+    const checklyConfigData = recast.print(checklyAst, { tabWidth: 2 }).code
+    const writeDir = path.resolve(path.dirname(configFile.fileName))
+    await reWriteChecklyConfigFile(checklyConfigData, configFile.fileName, writeDir)
+    this.style.actionSuccess()
+    return
+
+  }
+
+  private static async getTestCommand(directoryPath: string, input: string): Promise<string| undefined> {
+    const packageManager = await detectPackageManager(directoryPath)
+    if (!packageManager) {
+      throw new Error('Unable to detect package manager. Please ensure you are in a valid Node.js project directory.')
+    }
```

Comment:
You can remove this check - `detectPackageManager()` always returns npm as the final fallback, so it will never return an undefined value.